nox.py

```diff
@@ -83,19 +70,17 @@ def lint(session):
     session.interpreter = 'python3.6'
     session.install('flake8')
     session.install('.')
-    session.run(
-        'flake8',
-        '--exclude=opencensus/trace/exporters/gen/opencensus/',
-        'opencensus/')
+    session.run('flake8', '--max-line-length=80',
```

Comment:
I think it's better to set the line length to 79 in the formatter than force flake8 to match the formatter. This seems like putting the cart before the horse.

Reply:
We would like to use google style guide:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length